lib/course_revision_updater.rb

```diff
@@ -7,21 +7,21 @@ class CourseRevisionUpdater
   ###############
   # Entry point #
   ###############
-  def self.import_revisions(update_cs, all_time:)
-    course = update_cs.course
+  def self.import_revisions(course, all_time:, update_cs: nil)
+    course = course
     return if course.students.empty?
-    new(update_cs).update_revisions_for_relevant_wikis(all_time)
+    new(course, update_cs: update_cs).update_revisions_for_relevant_wikis(all_time)
     ArticlesCourses.update_from_course(course)
   end
 
-  def initialize(update_cs)
+  def initialize(course, update_cs: nil)
```

Comment:
All the cases of `update_cs` need a better name.

Reply:
I have renamed all of them to `update_object`